hazelcast/src/test/java/com/hazelcast/map/impl/tx/TxnMapNearCacheSerializationCountTest.java

```diff
@@ -75,9 +75,9 @@
                 {new int[]{1, 0, 0}, new int[]{0, 1, 1}, BINARY, BINARY,},
                 {new int[]{1, 0, 0}, new int[]{0, 1, 1}, BINARY, OBJECT,},
 
-                {new int[]{1, 1, 1}, new int[]{2, 1, 1}, OBJECT, null,},
-                {new int[]{1, 1, 1}, new int[]{2, 1, 1}, OBJECT, BINARY,},
-                {new int[]{1, 1, 1}, new int[]{2, 1, 1}, OBJECT, OBJECT,},
+                {new int[]{1, 1, 1}, new int[]{1, 1, 1}, OBJECT, null,},
```

Comment:
There is one deserialization less, due to the `PutBackupOperation` not being executed.